build_manpages/build_manpages.py

```diff
@@ -78,10 +102,12 @@ def initialize_options(self):
 
 
     def finalize_options(self):
-        if not self.manpages:
-            raise DistutilsOptionError('\'manpages\' option is required')
-
-        self.manpages_data = parse_manpages_spec(self.manpages)
+        manpages = self.manpages
+        if not manpages:
+            manpages = get_pyproject_settings()
```

Comment:
These three lines can be done like `manpages = self.manpages or `get_pyproject_settings()`

Reply:
Done.